app/user-setup/index.tsx

```diff
@@ -0,0 +1,215 @@
+            await setUpUser(data);
+            router.back();
+        } catch (error) {
+            console.error("Setup Failed:", error);
+        } finally {
+            setIsLoading(false);
+        }
+    }
+
+    return (
+        <View className="flex-1 bg-white">
+            <View className="flex-row justify-start">
+                <Text className="my-4 text-lg font-semibold">Setup User </Text>
+                <Text className="my-4 text-lg color-gray-400">
+                    (Step 1 out of 1)
+                </Text>
+            </View>
+            <Divider />
+            <View className="my-5 flex items-center">
+                <Text className="m-4 text-2xl font-extrabold">Avatar</Text>
+                <View className="aspect-square w-1/2 items-center overflow-hidden rounded-full">
+                    <Image
+                        source={{ uri: imageURI }}
+                        className="h-full w-full"
+                        resizeMode="cover"
+                    />
+                </View>
+            </View>
+
+            <View className="flex-row justify-center">
+                <TouchableOpacity
+                    className="m-2 rounded border px-4 py-2"
+                    onPress={openImageLibrary}
+                >
+                    <Text>Upload new image</Text>
+                </TouchableOpacity>
+                <TouchableOpacity
+                    className="m-2 rounded border px-4 py-2"
+                    onPress={openCamera}
+                >
+                    <Text>Take a picture</Text>
+                </TouchableOpacity>
+            </View>
+            <View className="mt-5">
+                <Text className="my-2 text-xl font-extrabold">
+                    Display name
+                </Text>
+                <Controller
+                    control={control}
+                    name="name"
+                    render={({ field: { onChange, value } }) => (
+                        <TextInput
+                            className="border-hairline h-14 rounded-md border-black px-3 py-1.5 text-lg text-gray-900"
+                            placeholder="Enter Display Name"
+                            onChangeText={onChange}
+                            value={value}
+                        />
+                    )}
+                />
+                {errors.name && (
+                    <Text className="mt-1 text-red-500">
+                        {errors.name.message}
+                    </Text>
+                )}
+            </View>
+            <View className="flex-col">
+                <View className="mb-2 mt-4 flex-row justify-start">
+                    <Text className="text-xl font-extrabold">Campus </Text>
+                    <Text className="text-xl color-gray-400">
+                        (preferred location)
+                    </Text>
+                </View>
+                {CampusEnum.options.map((campus, _) => {
```

Comment:
no need to have `_`

can simplify to:
`CampusEnum.options.map((campus) => ....`